app/views/shared/_messages.html.haml

```diff
@@ -1,4 +1,4 @@
 - flash.each do |name, msg|
-  %div(class="alert alert-#{name == :notice ? "success" : "danger" }")
+  %div(class="alert alert-#{name == "error" ? "danger" : "info" }")
```

Comment:
I think you can just do `name` instead of  `name == "error" ? "danger" : "info"`. All the time you are using key symbol that respects Bootstrap 3 alerts :).